src/components/Tabs.js

```diff
@@ -0,0 +1,204 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const SimpleSelect = require('./SimpleSelect');
+const StyleConstants = require('../constants/Style');
+
+const Tabs = React.createClass({
+  propTypes: {
+    activeTabStyles: React.PropTypes.object,
+    brandColor: React.PropTypes.string,
+    onTabSelect: React.PropTypes.func.isRequired,
+    selectedTab: React.PropTypes.number,
+    showBottomBorder: React.PropTypes.bool,
+    tabs: React.PropTypes.array.isRequired,
+    useTabsInMobile: React.PropTypes.bool
```

Comment:
This was something that @derek-boman wanted because he said that the normal case is that we would want the `menu` in mobile but there is one case where we would not want the `menu` in mobile but we would want to use `tabs` in mobile instead.